src/Pages/SignUp/SignUp.scss

```
@@ -1,2 +1,212 @@
 @import "../../Styles/common.scss";
-@import "../../Styles/reset.scss";
+
+.SignUp {
+  ::placeholder {
+    color: rgb(214, 214, 214);
+  }
+  .kakaoWrap {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+    .kakaoHead {
```

```suggestion
    padding-top: 50px;
    
    .kakaoHead {
```
새로운 셀렉터 앞에서는 한줄 띄워주세요!

---

네엡 수정 완료했습니다 !!